bluepyopt/ephys/simulators.py

```diff
@@ -16,11 +16,14 @@ def __init__(self, dt=None, cvode_active=True, cvode_minstep=None):
 
         import imp
         import ctypes
-
-        hoc_so = os.path.join(imp.find_module('neuron')[1] + '/hoc.so')
-
-        nrndll = ctypes.cdll[hoc_so]
-        ctypes.c_int.in_dll(nrndll, 'nrn_nobanner_').value = 1
+        import platform
+
+        if platform.system()!='Windows':
+            # hoc.so does not exist on NEURON Windows
```

Comment:
> is there a construct to disable the Neuron banner, similar to the code below, that works on Windows ?
 On launch you can use the option -nobanner
(see all options with:     nrniv -h )